editor.planx.uk/src/pages/FlowEditor/components/Flow/components/Node.tsx

```diff
@@ -128,7 +128,7 @@ const Node: React.FC<any> = (props) => {
       return (
         <Question
           {...allProps}
-          text={[node?.data?.flagSet ?? "Result"].join("")}
+          text={`Result - ${node?.data?.flagSet ?? "Result"}`}
```

Comment:
nit: this could result in `Result - Result` currently, instead I think we want something like this? 
```
text={node?.data?.flagSet ? `Result - ${node.data.flagSet}` : `Result`}
```